Multiplicity.Packets/WorldInfo.cs

```diff
@@ -102,14 +107,27 @@ public class WorldInfo : TerrariaPacket
         /// </summary>
         public byte EventInfo2 { get; set; }
```

Comment:
Hmm, I mentioned using the bitflags in my previous comment about the player controls. It could be useful to have a Flags enum for things like this as well. This isn't strictly relevant to this PR, but this could be added later on. Let's keep it in mind.